code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -418,10 +407,10 @@
 	overdose_threshold = 30
 
 /datum/reagent/medicine/omnizine/on_mob_life(mob/living/M)
-	M.adjustToxLoss(-0.5*REM, 0)
-	M.adjustOxyLoss(-0.5*REM, 0)
-	M.adjustBruteLoss(-0.5*REM, 0)
-	M.adjustFireLoss(-0.5*REM, 0)
+	M.adjustToxLoss(-1*REM, 0)
+	M.adjustOxyLoss(-1*REM, 0)
+	M.adjustBruteLoss(-1*REM, 0)
+	M.adjustFireLoss(-1*REM, 0)
```

Comment:
Why make Omnizine even better?

Reply:
i had a twenty minute argument with jessie where they argued in favor of making botany have stronger healing chemicals in plants and removing chemistry entirely so this was a compromise where i would see if people would do botany more over chemistry if the plant healing chems were better